src/lib/utils/gasLimitHelper.ts

```diff
@@ -1,17 +1,23 @@
 import { GasGroup, MAX_GAS_PER_GROUP } from '~app/common/config/gasLimits';
 
-const calculateTenPercentIncrease = (value: number): number =>  {
-    return Math.floor(value * 1.2);
+const USE_HARDCODED_GAS_LIMIT = 'USE_HARDCODED_GAS_LIMIT';
+
+const calculateTenPercentIncrease = (value: number): number | null =>  {
+      const result = window.localStorage.getItem(USE_HARDCODED_GAS_LIMIT);
+      if ( result && JSON.parse(result)) {
+          return Math.floor(value * 1.2);
+      }
+    return null;
```

Comment:
Please implement feature flags logic the same as you can see it in explorer UI.
You can simply copy-paste it from there and use your own flags.
It would be better also to migrate to that logic all the rest of the flags that we have in this webapp.
Why this is important: to have all the flags and their logic in one place

Reply:
done